### ISSUE TITLE
easyplace/material list not recognizing items with lore on them

### ISSUE THREAD

**Issue by RimonsVirginity** [maruohon/litematica#675]
is there a way to ignore the item lore and nbt data and make it only check the material? maybe an option that can be added

**Comment by maruohon** [comment]
Looks like there isn't an option for it currently. I should probably just change it to ignore NBT data in all cases(?)

**Comment by RimonsVirginity** [comment]
yes please! ignore nbt data and lore, just check for material only.

**Comment by maruohon** [comment]
The latest 1.19.2, 1.19.3 and 1.19.4 versions now ignore NBT data on the items for the material list available items count.

However I didn't change the pick bock and Easy Place stuff yet, because those changes should then also happen partially on malilib's side and I didn't feel like doing such a change at this time. So that change will probably come later via the main development code from 1.12.2.
